systemtests/volsupervisor_test.go

```diff
@@ -1,12 +1,61 @@
 package systemtests
 
 import (
+	"fmt"
 	"strings"
 	"time"
 
+	log "github.com/Sirupsen/logrus"
 	. "gopkg.in/check.v1"
 )
 
+func (s *systemtestSuite) TestVolsupervisorSnapLockedVolume(c *C) {
+	if !cephDriver() {
+		c.Skip("Only ceph supports snapshots")
+		return
+	}
+
+	_, err := s.uploadIntent("policy1", "snaplockedvol")
+	c.Assert(err, IsNil)
+
+	volName := genRandomVolume()
+	fqVolName := fqVolume("policy1", volName)
+	c.Assert(s.createVolume("mon0", fqVolName, nil), IsNil) // locked volume
+
+	time.Sleep(4 * time.Second)
```

Comment:
This sleep is to ensure that no snaps has been created when the volume is not mounted.